### PR TITLE
Fixes #1280

### LINKED ISSUE #1280: unable to change password.
**Summary:**
Unable to change password if language is other than English

**Steps to reproduce:**
Settings-->Language-->change Language-->Click on change Password 

unable to change
**Expected behaviour:**

Password must be changed 

**Observed behaviour:**
When the application language is set to language other than English, we cannot change the password

**Device and Android version:**
Vivo Y51L

**Screenshots:**
ezgif com-video-to-gif (4)

### PR DESCRIPTION
**Pull request by divyank00**
Fixes #1280

Please Add Screenshots If there are any UI changes.
Desktop_20200120012259

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by divyank00** [comment]
@devansh-299 Can you review this?
Thank you!

**Comment by devansh-299** [comment]
> 
> 
> @devansh-299 Can you review this?
> Thank you!

@divyank00 I guess you are working over your commit only . Please first reset you branch (preferably create a new branch for this PR) to HEAD then do the changes for this PR

**Comment by divyank00** [comment]
I made a new branch, but still

File: app/src/main/java/org/mifos/mobile/ui/fragments/SavingsMakeTransferFragment.java [hunk 1]
```diff
@@ -397,6 +399,10 @@ public void onNothingSelected(AdapterView<?> parent) {
      */
     @OnClick(R.id.btn_pay_to)
     public void payToSelected() {
+        if (toAccountOption == null) {
```
Comment (laxyapahuja) [inline app/src/main/java/org/mifos/mobile/ui/fragments/SavingsMakeTransferFragment.java:402]: not related to this issue.

File: app/src/main/java/org/mifos/mobile/ui/fragments/SavingsMakeTransferFragment.java [hunk 2]
```diff
@@ -414,6 +420,9 @@ public void payToSelected() {
      */
     @OnClick(R.id.btn_pay_from)
     public void payFromSelected() {
+        if (fromAccountOption == null) {
```
Comment (laxyapahuja) [inline app/src/main/java/org/mifos/mobile/ui/fragments/SavingsMakeTransferFragment.java:423]: not related to this issue

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <resources>
-    <string name="app_name">Mifos Mobile</string>
+    <string name="app_name" translatable="false">Mifos Mobile</string>
```
Comment (laxyapahuja) [inline app/src/main/res/values/strings.xml:2]: unnecessary

File: build.gradle [hunk 4]
```diff
@@ -38,7 +38,7 @@ ext {
     buildToolsVersion = '28.0.3'
 
     // App dependencies
-    supportLibraryVersion = '1.1.0'
+    supportLibraryVersion = '1.0.0'
```
Comment (laxyapahuja) [inline build.gradle:41]: what does it do exactly?

File: app/src/main/java/org/mifos/mobile/ui/fragments/ThirdPartyTransferFragment.java [hunk 5]
```diff
@@ -296,6 +298,9 @@ public void showBeneficiaryList(List<Beneficiary> beneficiaries) {
      */
     @OnClick(R.id.btn_pay_from)
     public void payFromSelected() {
+        if (fromAccountOption == null) {
```
Comment (laxyapahuja) [inline app/src/main/java/org/mifos/mobile/ui/fragments/ThirdPartyTransferFragment.java:301]: unnecessary

File: app/src/main/java/org/mifos/mobile/ui/fragments/ThirdPartyTransferFragment.java [hunk 6]
```diff
@@ -320,6 +325,10 @@ public void payFromSelected() {
      */
     @OnClick(R.id.btn_pay_to)
     public void payToSelected() {
+        if (beneficiaryAccountOption == null) {
```
Comment (laxyapahuja) [inline app/src/main/java/org/mifos/mobile/ui/fragments/ThirdPartyTransferFragment.java:328]: unnecessary

File: app/src/main/java/org/mifos/mobile/ui/fragments/ThirdPartyTransferFragment.java [hunk 7]
```diff
@@ -231,22 +231,24 @@ public void reviewTransfer() {
...
-        transferPayload.setToAccountId(beneficiaryAccountOption.getAccountId());
-        transferPayload.setToClientId(beneficiaryAccountOption.getClientId());
-        transferPayload.setToAccountType(beneficiaryAccountOption.getAccountType().getId());
-        transferPayload.setTransferDate(transferDate);
-        transferPayload.setTransferAmount(Double.parseDouble(etAmount.getText().toString()));
-        transferPayload.setTransferDescription(etRemark.getText().toString());
-
-        ((BaseActivity) getActivity()).replaceFragment(TransferProcessFragment.
-                newInstance(transferPayload, TransferType.TPT), true, R.id.container);
+        if (fromAccountOption != null && beneficiaryAccountOption != null) {
```
Comment (laxyapahuja) [inline app/src/main/java/org/mifos/mobile/ui/fragments/ThirdPartyTransferFragment.java:234]: unnecessary

File: app/src/main/java/org/mifos/mobile/ui/fragments/SavingsMakeTransferFragment.java [hunk 8]
```diff
@@ -216,23 +216,25 @@ void reviewTransfer() {
...
-        transferPayload.setToAccountType(toAccountOption.getAccountType().getId());
-        transferPayload.setTransferDate(transferDate);
-        transferPayload.setTransferAmount(Double.parseDouble(etAmount.getText().toString()));
-        transferPayload.setTransferDescription(etRemark.getText().toString());
-        transferPayload.setFromAccountNumber(fromAccountOption.getAccountNo());
-        transferPayload.setToAccountNumber(toAccountOption.getAccountNo());
-
-        ((BaseActivity) getActivity()).replaceFragment(TransferProcessFragment.
-                newInstance(transferPayload, TransferType.SELF), true, R.id.container);
+        if (fromAccountOption != null && toAccountOption != null) {
```
Comment (laxyapahuja) [inline app/src/main/java/org/mifos/mobile/ui/fragments/SavingsMakeTransferFragment.java:219]: unnecessary

**Review by laxyapahuja — COMMENTED** [review]
It seems like your branch has your old commits too which means you have not created a branch correctly. Always create a branch with upstream master as the base branch.

File: app/src/main/java/org/mifos/mobile/ui/fragments/SavingsMakeTransferFragment.java (hunk 1 above)
Comment (divyank00) [reply app/src/main/java/org/mifos/mobile/ui/fragments/SavingsMakeTransferFragment.java:402]: Other issue

File: build.gradle (hunk 4 above)
Comment (divyank00) [reply build.gradle:41]: The recycler view in Account Section is damaged.

**Comment by laxyapahuja** [comment]
I think you'll have to close this PR.

**Comment by ShivangiSingh17** [comment]
@divyank00 please close this PR as the code changes made are not supporting the issue you want to address. Also it contains previously committed changes.

**Review by ShivangiSingh17 — COMMENTED** [review]
Close the PR.

**Pull request closed without merging**